aioresponses/core.py

```diff
@@ -201,6 +243,8 @@ def __init__(self, **kwargs):
                              side_effect=self._request_mock,
                              autospec=True)
         self.requests = {}
+        self._calls = CallList()
```

Comment:
We already have `requests` collection where we "keep" some call breadcrumbs. 
Maybe we should use/extend that in order to implement those new methods.

Reply:
Hey @pnuckowski :wave: :smiley: 
Current implementation has:
* `requests` as a dictionary of `{'key': [RequestCall(...),]}`
* `_responses` as a dictionary of `{'key': ClientResponse(...)}`
* `_matches` as a list of `RequestMatch(...)` but the items are deleted after match

My idea here was to keep `RequestMatch`es somewhere for later evaluation.

I'm afraid to change behavior of existing properties and `calls` makes it more straightforward to me. My only idea is to make them more private... What would you recommend here?

Thanks!!